packages/uui-text-copy/lib/UUITextCopyEvent.ts

```diff
@@ -0,0 +1,21 @@
+import { UUIEvent } from '@umbraco-ui/uui-base/lib/events';
+import { UUITextCopyElement } from './uui-text-copy.element';
+
+interface UUITextCopyEventInit extends EventInit {
+  detail?: { text: string };
+}
+
+export class UUITextCopyEvent extends UUIEvent<
+  { text: string },
+  UUITextCopyElement
+> {
+  public static readonly COPIED: string = 'copied';
+  public static readonly COPYING: string = 'copying';
+
+  constructor(evName: string, eventInit: UUITextCopyEventInit = {}) {
+    super(evName, {
+      ...{ bubbles: true },
```

Comment:
Any reason for this event to bubbles? I think it shouldn't bubble, but maybe you have a reason?

Reply:
I was probably doing it for the example/story to allow people to catch the event @nielslyngsoe 
If you think I should remove it, I will do